config/project/entryTypes/event--12c6ecb1-eb21-40a2-aca1-217b86047c44.yaml

```diff
@@ -0,0 +1,292 @@
+            instructions: null
+            label: Types
+            providesThumbs: false
+            required: false
+            tip: null
+            type: craft\fieldlayoutelements\CustomField
+            uid: 2cc09c24-ea29-47ac-aa51-21495d792080
+            userCondition: null
+            warning: null
+            width: 50
+          -
+            dateAdded: '2025-02-26T02:38:49+00:00'
+            elementCondition: null
+            fieldUid: 69339afc-d4f1-440b-a7ad-7da2ccb77972 # Image
+            handle: cover
+            includeInCards: false
+            instructions: null
+            label: 'Cover Image'
+            providesThumbs: false
+            required: false
+            tip: null
+            type: craft\fieldlayoutelements\CustomField
+            uid: 04ba682b-6348-48de-833f-c03c6dd10b6b
+            userCondition: null
+            warning: null
+            width: 50
+          -
+            dateAdded: '2025-02-26T02:38:49+00:00'
+            elementCondition: null
+            fieldUid: bb0a4a47-dcbd-41a8-9bbb-82341295f5ff # Date & Time
+            handle: startDateTime
+            includeInCards: false
+            instructions: null
+            label: 'Start Date'
+            providesThumbs: false
+            required: false
+            tip: null
+            type: craft\fieldlayoutelements\CustomField
+            uid: e30e5ed4-2a93-4299-927a-7993d8dc68b3
+            userCondition: null
+            warning: null
+            width: 50
+          -
+            dateAdded: '2025-02-26T02:38:49+00:00'
+            elementCondition: null
+            fieldUid: bb0a4a47-dcbd-41a8-9bbb-82341295f5ff # Date & Time
+            handle: endDateTime
+            includeInCards: false
+            instructions: null
+            label: 'End Date'
+            providesThumbs: false
+            required: false
+            tip: null
+            type: craft\fieldlayoutelements\CustomField
+            uid: 7f13ccc6-9d8e-4cbb-b64c-7821ab39219b
+            userCondition: null
+            warning: null
+            width: 50
+          -
+            dateAdded: '2025-02-26T02:40:53+00:00'
+            elementCondition: null
+            fieldUid: afc7f0bc-1b97-4841-a1d2-5b0385a7ca35 # Plain Text
+            handle: location
+            includeInCards: false
+            instructions: null
+            label: Location
+            providesThumbs: false
+            required: false
+            tip: null
+            type: craft\fieldlayoutelements\CustomField
+            uid: c4c78f42-c653-4f93-8661-742ba26442c5
+            userCondition: null
+            warning: null
+            width: 50
+          -
+            dateAdded: '2025-02-26T02:38:49+00:00'
+            elementCondition: null
+            fieldUid: d356acc9-4ecf-488d-a89d-2857490bb576 # Hyperlink
+            handle: registrationLink
+            includeInCards: false
+            instructions: null
+            label: 'Registration Link'
```

Comment:
The URL might not always be for “registration” only, right? Sometimes it might just point to a more canonical event page/microsite and not directly to the registration link.